packages/react-components/src/shared/src/slots.tsx

```diff
@@ -52,15 +52,18 @@ export function useRawSlots(children: ReactNode, slots: string[]) {
     return useMemo(() => getRawSlots(children, slots), [children, slots]);
 }
 
-interface Slots {
+interface RequiredDefaultSlot {
```

Comment:
Hmm why this rename?

Don't like my names to be an Concatenation of the type props.. If I add a "toto" props will I rename this to RequiredDefaultTotoSlot?

Reply:
Could be named maybe SlotsDefinition

Reply:
Maybe `SlotOptions` ?